tasks/src/sentence-similarity/data.ts

```diff
@@ -66,25 +66,25 @@ const taskData: TaskData = {
 	libraries: TASKS_MODEL_LIBRARIES["sentence-similarity"],
 	metrics:   [
 		{
-			description: "The reciprocal rank is a measure used to rank the relevancy of documents given a set of documents. Reciprocal Rank is the reciprocal of the rank of the document retrieved, meaning, if the rank is 3, the Reciprocal Rank is 0.33. If the rank is 1, the Reciprocal Rank is 1.",
+			description: "Reciprocal ranking is a measure used to rank the relevance of documents given a set of documents. The Reciprocal Rank is the reciprocal of the retrieved document rank, that is, if the rank is 3, the Reciprocal Rank is 0.33. If the rank is 1, the Reciprocal Rank is 1.",
 			id:          "Mean Reciprocal Rank",
 		},
 		{
-			description: "The similarity of embeddings is evaluated mainly on cosine similarity. It’s calculated as the cosine of the angle between two vectors. It is particularly useful when your documents do not have the same length.",
```

Comment:
The metric is called "cosine similarity" directly so I wouldn't change that.

Reply:
To build on top of that, similarity can be measured in different ways, with cosine being one of them; that's why we mention cosine similarity